internal/p4constants/p4constants.go

```diff
@@ -111,6 +112,125 @@ const (
 	PacketMetaPacketOut uint32 = 75327753
 	PacketMetaPacketIn  uint32 = 80671331
 	// Meters
-	MeterPreQosPipeAppMeter     uint32 = 338231090
-	MeterPreQosPipeSessionMeter uint32 = 347593234
+	MeterPreQosPipeAppMeter         uint32 = 338231090
+	MeterSizePreQosPipeAppMeter     uint64 = 1024
+	MeterPreQosPipeSessionMeter     uint32 = 347593234
+	MeterSizePreQosPipeSessionMeter uint64 = 1024
+)
+
+var (
+	TableIDToNameMap = map[uint32]string{
```

Comment:
In principle, we should have immutable maps and lists here. In native go, maps and lists cannot be declared `const`. We could use this, though : https://github.com/benbjohnson/immutable